03Lesson/01Project/css/style.css

```diff
@@ -1 +1,52 @@
-
+body {
+  font-family: Arial;
+}
+.dev_1 {
+  border: 1px
+  solid red;
+}
+.dev_2 {
+  border: 1px
+  solid purple;
+}
+.blue {
+  background-color: #0080ff
+}
+.bluetext {
+  color: #0080ff
+}
+.grey {
+  background-color: #f2f2f2
+}
+.tinytext {
+  font-size: 12px;
+}
+.split {
+  float: left;
+}
+.split2 {
+  float: right;
+}
+section {
+  height: auto;
+  width: auto;
+  padding: 10px;
+}
+h1 {
+  color: #ffffff
+}
+a {
+  color: #ffffff
+}
+table {
+  width: 100%;
+}
+th {
+  height: 50%;
+}

```

Comment:
Duplicate definition of styles for `table` and `th`. Better remove one, otherwise you might change styles and wonder why they are not being applied...